portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeErcForCategories.testcase

```diff
@@ -0,0 +1,65 @@
+@component-name = "portal-headless-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless Frontend Infrastructure";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given an asset library is created") {
+			JSONDepot.addDepot(depotName = "Test Depot Name");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONDepot.deleteDepot(depotName = "Test Depot Name");
+		}
+	}
+
+	@priority = "4"
+	test CanRecieveCorrectErcValueInCategory {
+		property portal.acceptance = "true";
+
+		task ("Given a vocabulary with assetLibrary") {
```

Comment:
Do you think is that ok to move the "Given a vocabulary is created in assetLibrary" into SetUp? As far as I know, we did the same action for structuredContent in assetLibrary.

Reply:
This involves the transmission of parameter values, which can be temporarily not modified. If there is a major update in the future, it can be modified together